diff --git a/dist/index.mjs b/dist/index.mjs
index 28cbe2d5b73463e933235462036c6ce5f49dcd7e..6665b21eec02104c85bb91b4e3555c376e20521c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -31,7 +31,8 @@ const sveltekitBundleAnalysisPlugin = ({
         output.originalBundleName,
         options.dir,
         options.format,
-        options.name
+        // https://github.com/codecov/codecov-javascript-bundler-plugins/issues/252
+        // options.name
       );
       output.unlockBundleName();
       output.setBundleName(name);
