diff --git a/node_modules/@codecov/sveltekit-plugin/dist/index.mjs b/node_modules/@codecov/sveltekit-plugin/dist/index.mjs
index 28cbe2d..6665b21 100644
--- a/node_modules/@codecov/sveltekit-plugin/dist/index.mjs
+++ b/node_modules/@codecov/sveltekit-plugin/dist/index.mjs
@@ -31,7 +31,8 @@ const sveltekitBundleAnalysisPlugin = ({
         output.originalBundleName,
         options.dir,
         options.format,
-        options.name
+        // https://github.com/codecov/codecov-javascript-bundler-plugins/issues/252
+        // options.name
       );
       output.unlockBundleName();
       output.setBundleName(name);
